pypasta/MailCharacteristics.py

```diff
@@ -171,12 +171,31 @@ def _is_from_bot(self):
         if self.is_next and 'sfr@canb.auug.org.au' in email:
             return True
 
+        # Github Bot
+        if 'noreply@github.com' in email:
+            return True
+
+        # Buildroot's daily results bot
+        if '[autobuild.buildroot.net] Daily results' in subject:
+            return True
+
+        # Buildroot's daily results bot
+        if 'oe-core cve metrics' in subject.lower():
+            return True
+
         return False
 
     def _integrated_correct(self, repo, maintainers_version):
+        if self.first_upstream in repo:
+            upstream = repo[self.first_upstream]
+            self.committer = upstream.committer.name.lower()
```

Comment:
Wollen wir hier vielleicht die Mailadresse ausleiten? Die ist aussagekräftiger, oder?

Reply:
Okay, I remember that - for some reason - we talked about this change, but I don't understand at the moment why we did this. Could you please factor this change out to an own commit with an explanation why we do this? This change is preparatory work for supporting other projects, and not related to the introduction of Jailhouse.

Reply:
We need this to get the committer of patches in projects without a maintainer.
Without this change, we would return from this function before we get this information.
The information about the committer however is relevant and valid for all projects.